static/app/components/profiling/FrameStack/frameStack.tsx

```diff
@@ -148,7 +165,29 @@ const FrameStack = memo(function FrameStack(props: FrameStackProps) {
             {t('Collapse recursion')}
           </FrameDrawerLabel>
         </li>
-        <li style={{flex: '1 1 100%', cursor: 'ns-resize'}} onMouseDown={onMouseDown} />
+        <li
+          style={{
+            flex: '1 1 100%',
+            cursor:
+              flamegraphPreferences.layout === 'table_bottom' ? 'ns-resize' : undefined,
+          }}
+          onMouseDown={
+            flamegraphPreferences.layout === 'table_bottom' ? onMouseDown : undefined
```

Comment:
i notice we keep repeating the assertion `flamegraphPreferences.layout === 'table_bottom'`; why not just assign this to a variable to DRY it up

Reply:
@eliashussary I'm not a very big fan of DRY when it comes to semantics vs syntax. You could write `const isTableLayoutBottom = flamegraphPreferences.layout === 'table_bottom'`, but that is not very valuable in itself and it represents another abstraction to lookup when reading the code.